addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1305,11 +1309,15 @@ export class Rtc extends Record {
                 Object.assign(this.state, {
                     sourceScreenStream: sourceStream,
                     screenTrack: outputTrack,
+                    screenAudioTrack: screenAudioTrack,
                     sendScreen: Boolean(outputTrack),
                 });
                 break;
             }
         }
+        if (this.state.screenAudioTrack) {
+            this.resetAudioTrack({ force: true });
```

Comment:
I wouldn't call `resetAudioTrack()` here, it's doing a lot of unnecessary things, I think that there should be a new function that takes care of merging `this.state.screenAudioTrack` and `this.state.audioTrack` and updating the network, called both here and in`resetAudioTrack()`.